Imager.js

```diff
@@ -142,30 +143,36 @@
           });
         }
 
-        if (elements) {
-            this.divs = applyEach(elements, returnFn);
-            this.selector = null;
-        }
-        else {
-            this.divs = applyEach(doc.querySelectorAll(this.selector), returnFn);
-        }
-
+        this.divs = [];
+        this.add(elements || this.selector);
         this.ready(opts.onReady);
-        this.changeDivsToEmptyImages(this.divs);
 
-        setTimeout(function(){
+        setTimeout(function () {
             self.init();
         }, 0);
-    }
+    };
+
+    Imager.prototype.add = function (elementsOrSelector) {
+
+        elementsOrSelector = elementsOrSelector || this.selector;
```

Comment:
The problem here is with post-loading elements, i.e. ajax, as this could introduce new elements with a class of ```delayed-image-load``` i.e.

```
var imgr = new Imager(); // default selector is 'delayed-image-load'
...
// ajax some elements into the DOM with class of 'delayed-image-load'
...
imgr.add(); // doesn't pick up elements unless we allow default selector to apply
```

This would be  a workaround for users but doesn't seem right:

```
imgr.add(imgr.selector);
```

Allowing a "no argument" version of ```add``` is the best possible solution I can think of.